docs/README.ubuntu

```diff
@@ -113,6 +113,16 @@ We supply a Makefile in lib/taglib to make it easy to install into /usr/local.
 or use prepackaged from the Kodi PPA.
 
     $ sudo apt-get install libtag1-dev
+	
+For >= 14.04
```

Comment:
we need crossguid on all platforms and linux versions, but the PPA only provides it for >= 14.04.
Please also make the PPA install method the "default", i.e. put it before the manual install.